src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -254,6 +260,107 @@ public void SarifTransformerTests_ToCurrent_OneRunWithFiles()
             fileData.FileLocation.Uri.OriginalString.Should().Be("file:///docs/intro.docx");
             fileData.Contents.Should().NotBeNull();
             fileData.Contents.Binary.Should().BeNull();
+            fileData.Properties.Should().BeNull();
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWitRules()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""rules"": {
+                        ""C2001"": {
+                          ""id"": ""C2001"",
+                          ""shortDescription"": ""A variable was used without being initialized."",
+                          ""messageFormats"": {
+                            ""default"": ""Variable \""{0}\"" was used without being initialized.""
+                          },
+                          ""properties"": {
+                            ""some_key"": ""FoxForceFive""
+                          }
+                        },
+                        ""C2002"": {
+                          ""id"": ""C2002"",
+                          ""fullDescription"": ""Catfish season continuous hen lamb include dose copy grant."",
+                          ""configuration"": ""enabled"",
+                          ""defaultLevel"": ""error"",
+                          ""helpUri"": ""http://www.domain.com/rules/c2002.html""
+                        },
+                        ""C2003"": {
+                          ""id"": ""C2003"",
+                          ""name"": ""Rule C2003"",
+                          ""shortDescription"": ""Rules were meant to be broken."",
+                          ""fullDescription"": ""Rent internal rebellion competence biography photograph."",
+                          ""configuration"": ""disabled"",
+                          ""defaultLevel"": ""pass""
+                        }
+                      },
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            v2Log.Runs.Should().NotBeNull();
+            v2Log.Runs.Count.Should().Be(1);
+            v2Log.Runs[0].Resources.Should().NotBeNull();
```

Comment:
Let's use our shiny new `run` temp var here too.

Reply:
Ah missed that one

Reply:
Done